src/transforms.js

```diff
@@ -133,7 +133,14 @@ export const safeInsert = (content, position, tryToReplace) => tr => {
     : isNodeSelection(tr.selection)
     ? tr.doc.resolve($from.pos + 1)
     : $from;
-  const { parent, depth } = $insertPos;
+  const { parent } = $insertPos;
+
+  const setSelection = (node, pos, tr) => {
```

Comment:
Can we declare this outside of this function? Doesnt look like we need to re-create it on every insert.